test/nut/commands/force/org/org.nut.ts

```diff
@@ -0,0 +1,214 @@
+
+const concatString = (a: string, b: string): string => a + b;
+
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+
+describe('Org Command NUT', function async() {
+  this.timeout(Duration.minutes(10).milliseconds);
+  let session: TestSession;
+  let defaultUsername: string;
+  let aliasedUsername: string;
+  let defaultUserOrgId: string;
+  let aliasUserOrgId: string;
+  before(() => {
+    session = TestSession.create({
+      project: { name: 'forceOrgList' },
+      setupCommands: [
+        'sfdx force:org:create -f config/project-scratch-def.json --setdefaultusername --wait 10',
+        'sfdx force:org:create -f config/project-scratch-def.json --setalias anAlias --wait 10',
+      ],
+    });
+
+    if (isArray<AnyJson>(session.setup)) {
+      defaultUsername = getString(session.setup[0], 'result.username');
+      defaultUserOrgId = getString(session.setup[0], 'result.orgId');
+      aliasedUsername = getString(session.setup[1], 'result.username');
+      aliasUserOrgId = getString(session.setup[1], 'result.orgId');
+    }
+  });
+
+  after(async () => {
+    await session?.clean();
+  });
+
+  describe('List Orgs', () => {
+    it('should list all orgs', () => {
+      const result = execCmd('force:org:list --json');
+      expect(result.shellOutput.code).to.be.equal(
+        0,
+        concatString(result.shellOutput.stdout, result.shellOutput.stderr)
+      );
+      const listResult = asDictionary(get(result, 'jsonOutput.result'));
+      expect(listResult).to.have.property('nonScratchOrgs');
+      expect(listResult.nonScratchOrgs).to.have.length(1);
+      expect(listResult).to.have.property('scratchOrgs');
+      expect(listResult.scratchOrgs).to.have.length(2);
+      const nonScratchOrgs = asDictionary(listResult.nonScratchOrgs[0]);
+      const scratchOrgs = listResult.scratchOrgs as unknown[];
+      expect(scratchOrgs.map((scratchOrg) => getString(scratchOrg, 'username'))).to.deep.equal([
+        defaultUsername,
+        aliasedUsername,
+      ]);
+      expect(scratchOrgs.map((org) => asDictionary(org)).find((org) => org.username === defaultUsername)).to.include({
+        defaultMarker: '(U)',
+        isDefaultUsername: true,
+      });
+      expect(scratchOrgs.map((org) => asDictionary(org)).find((org) => org.username === aliasedUsername)).to.include({
+        alias: 'anAlias',
+      });
+      expect(nonScratchOrgs).to.include(
+        {
+          username: process.env.TESTKIT_HUB_USERNAME,
+          defaultMarker: '(D)',
+          isDevHub: true,
+          connectedStatus: 'Connected',
+        },
+        JSON.stringify(nonScratchOrgs)
+      );
+    });
+    it('should list orgs - skipconnectionstatus', () => {
+      const result = execCmd('force:org:list --skipconnectionstatus --json');
+      expect(result.shellOutput.code).to.be.equal(
+        0,
+        concatString(result.shellOutput.stdout, result.shellOutput.stderr)
```

Comment:
This does produce nicer messages from mocha on fails.  Does this belong in testkit so Pete doesn't have to boilerplate it?  @shetzel 

https://github.com/salesforcecli/cli-plugins-testkit/blob/a018e3f43572df1c9c927c341eea1ed393b0ef6a/src/execCmd.ts#L81